examples/networking/vlan_filter/README.md

```diff
@@ -0,0 +1,29 @@
+# VLAN Filter #
+This is eBPF application to parse VXLAN packets and then extracts encapsulated VLAN packets to monitor traffic from each VLAN. Extracted packet header fields can be stored in a file or sent to remote server via Apache Kafka messaging system.
+
+### Usage Example ###
+* $ sudo python data-plane-tracing.py
+
+Timestamp | Host Name  | Host IP   | IP Version   | Source Host IP   | Dest Host IP   | Source Host Port   | Dest Host Port   | VNI   | Source VM MAC  | Dest VM MAC  | VLAN ID  | Source VM IP   | Dest VM IP   | Protocol   | Source VM Port   | Dest VM Port   | Packet Length   |
+---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---
+ 1526506199548287 | Box1 | x.x.x.x  | 4 | x.x.x.x | x.x.x.x | 54836 | 4789 | 1 | fa:16:3e:ec:22:99 | fa:16:3e:1c:6f:2d | 116 | 192.168.116.35 | 192.168.116.6 | 6 | 1285 | 20302 | 1200
```

Comment:
The time stamp here is hard to interpret. Maybe use a time format similar to tcpdump?

Reply:
As you have suggested, I have changed epoch timestamp to human readable timestamp format.